stdlib/public/stubs/Stubs.cpp

```diff
@@ -327,6 +340,54 @@ extern "C" long double _swift_fmodl(long double lhs, long double rhs) {
     return fmodl(lhs, rhs);
 }
 
+#if defined(__APPLE__)
+SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_INTERFACE
+extern "C" char ** _swift_get_args(int *outArgLen) {
+  assert(outArgLen != nullptr);
+
+  *outArgLen = *_NSGetArgc();
+  return *_NSGetArgv();
+}
+#elif defined(__linux__) || defined(__CYGWIN__) || defined(__FreeBSD__)
+SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_INTERFACE
+extern "C" char ** _swift_get_args(int *outArgLen) {
+  assert(outArgLen != nullptr);
+  
+  FILE *cmdline = fopen("/proc/self/cmdline", "rb");
+  if (cmdline == nullptr) {
+    *outArgLen = 0;
+    return nullptr;
+  }
+  char *arg = nullptr;
+  size_t size = 0;
+  std::vector<char *> argvec;
+  while(getdelim(&arg, &size, 0, cmdline) != -1) {
+    argvec.push_back(arg);
```

Comment:
Both of those are purely `out` parameters and I can't find any precedence for resetting them.